e2e/rbd_helper.go

```diff
@@ -167,9 +168,25 @@ func createRBDStorageClass(
 		sc.MountOptions = append(sc.MountOptions, mOpt...)
 	}
 	sc.ReclaimPolicy = &policy
-	_, err = c.StorageV1().StorageClasses().Create(context.TODO(), &sc, metav1.CreateOptions{})
 
-	return err
+	timeout := time.Duration(deployTimeout) * time.Minute
+
+	return wait.PollImmediate(poll, timeout, func() (bool, error) {
+		_, err = c.StorageV1().StorageClasses().Create(context.TODO(), &sc, metav1.CreateOptions{})
+		if err != nil {
+			e2elog.Logf("error creating StorageClass %q in namespace %q: %v", sc.Name, sc.Namespace, err)
+			if isRetryableAPIError(err) {
+				return false, nil
+			}
+			if apierrs.IsNotFound(err) {
+				return false, nil
+			}
```

Comment:
during create operation you will never get notFound error isn't it?

Reply:
probably not?

Reply:
Even I think so in that case we can remove this one.

Reply:
and this too